recipe/meta.yaml

```
@@ -1,41 +1,51 @@
 {% set name = "scikit-build-core" %}
-{% set version = "0.6.0" %}
+{% set version = "0.6.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit_build_core-{{ version }}.tar.gz
-  sha256: 1bea5ed83610b367f3446badd996f2356690548188d6d38e5b93152df311a7ae
+  sha256: 392254a4ca7235c27a4be98cc24cd708f563171961ce37cff66120ebfda20b7a
 
 build:
-  noarch: python
   number: 0
+  skip: true  # [py<37]
 
 requirements:
   host:
     - hatch-vcs
     - hatchling
     - pip
-    - python >=3.7
+    - python
   run:
-    - python >=3.7
-    - exceptiongroup
-    - importlib-metadata
-    - importlib-resources >=1.3
+    - python
+    - exceptiongroup             # [py<311]
+    - importlib_metadata         # [py<38]
```

```suggestion
    - importlib-metadata         # [py<38]
```
Let's keep the original PyPI name if possible.

---

agree, but what's with the `-` and `_`?

for `importlib-metadata` I can see

there is: https://anaconda.org/anaconda/importlib-metadata
there is ALSO: https://anaconda.org/anaconda/importlib_metadata

for the next package `importlib-resources`

there is: https://anaconda.org/anaconda/importlib-resources
there is **NOT**: https://anaconda.org/anaconda/importlib_resources

there is: https://anaconda.org/conda-forge/importlib-resources
there is ALSO: https://anaconda.org/conda-forge/importlib_resources